### DISCUSSION TITLE
IPA Files

### DISCUSSION THREAD

**Discussion by LesesTrickshon** [ppy/osu#29843] [Ideas] [answered]
I have noticed that the latest versions of osu!lazer aren‘t also being released as IPA files. It would make the life of ios players way easier because they could just sideload osu!lazer instead of having to either wait a long time for a spot in the testflight beta or playing an extremly outdated version. Also this would probably not be too difficult to do since nearly all of the bit older osu!lazer releases came with also an ipa file.

**Comment by Joehuu** [comment, marked as answer]
iOS builds are currently broken, tracked in https://github.com/ppy/osu/issues/28977.
